modules/node_modules/@ncigdc/components/SSM.js

```diff
@@ -289,19 +271,16 @@ const Mutation = compose(
           ${sortedCancerDistData.length} cancer projects`}
         />
       </Column>
-      {gene && (
-        <Column style={{ ...styles.card, marginTop: '2rem' }}>
-          <ProteinLolliplotComponent
-            gene={gene}
-            loading={vizLoading}
-            transcripts={gene.transcripts.hits.edges.map(x => x.node)}
-            selectTranscript={setActiveTranscript}
-            data={lolliplotData}
-          />
-        </Column>
-      )}
+      <Column style={{ ...styles.card, marginTop: '2rem' }}>
+        <Lolliplot
+          mutationId={mutation.ssm_id}
+          geneId={canonicalGeneId}
```

Comment:
So this is the part causing the relay error. Looks like we can't have props that are the same name as relay variables.